doc/examples/reconst_ivim.py

```diff
@@ -0,0 +1,211 @@
+"""
+============================================================
+Intravoxel incoherent motion
+============================================================
+The intravoxel incoherent motion (IVIM) model describes diffusion
+and perfusion in the signal acquired with diffusion. The IVIM model
```

Comment:
"acquired with diffusion" => "acquired with a diffusion MRI sequence that contains multiple low b-values"